website/docs/execution-node/fee-recipient.md

```diff
@@ -1,83 +1,172 @@
 ---
 id: fee-recipient
-title: Setting an Address for Receiving Transaction Fees
+title: How to configure Fee Recipient on your client instance and/or beacon node
```

Comment:
How do we feel about normalizing the use of "How to...." in titles of docs that are how tos? Pros that I see:

- May improve discoverability via search when people type "how to _______" 
- May be a step in the direction of organizing our docs into "How Tos" vs "Concepts" vs "Walkthroughs" - how tos being a specific type of walkthrough. Current thinking on this topic -> https://www.notion.so/prysmaticlabs/Principles-Content-0b2944089c99400f8d1ee3138b24fb65#ff546df2fb964819abef67b45ca3e176

Reply:
Reverting based on how clumsy it feels when rendered in large font, and based on the current pattern being followed in this node's sidebar.